x/incentives/keeper/gauge.go

```diff
@@ -277,6 +344,8 @@ func (k Keeper) GetActiveGauges(ctx sdk.Context) []types.Gauge {
 	return k.getGaugesFromIterator(ctx, k.ActiveGaugesIterator(ctx))
 }
 
+// TODO: implement GetActiveGroupGauges
```

Comment:
We probably need to define what "Active Group Gauges" mean. 

For regular gauges Active mean: (curTime.After(gauge.StartTime) || curTime.Equal(gauge.StartTime)) && (gauge.IsPerpetual || gauge.FilledEpochs < gauge.NumEpochsPaidOver)

Reply:
curious question: Are we trying to mimic GroupGauges to have Gauge like behavior? for ex: have upcomingGroupGauge, ActiveGroupGauge & finishedGroupGauge?

Reply:
Yes, since between internal and external incentives we'll want to cover the full breadth of these states